docs/getting_started/why_use_GTFS.md

```diff
@@ -1,17 +1,39 @@
 
 ## Open Data means more opportunity and choices
 
-GTFS is an Open Standard. This means that agencies can make information available using any of the many tools which already support GTFS (including simple text editing using a text editor or a spreadsheet). Open standards lead to the creation of data that can be easily shared. A feed is simply the collection of text files that describe a service, hosted online at a permalink that’s publicly available. The same feed can be used by Google, Apple, Transit App, Open Trip Planner, and even apps created by riders. Anyone who wants to provide accurate and up-to-date transit information can use a GTFS feed to do so. 
+GTFS is an Open Standard. This means that agencies can make information available using any of the tools which already support GTFS (including simple text editing or a spreadsheet). Open standards lead to the creation of data that can be easily shared. The feeds can be used by trip planners such as Google, Apple, Transit App, Open Trip Planner, and even apps created by riders. Anyone who wants to provide accurate and up-to-date transit information can use GTFS feed to do so.
 
 Some riders like to use different apps depending on their needs—having GTFS lets riders choose what trip planning app suits them best. Some apps may be more accessible or better at providing information for riders with disabilities, some may be simpler and easier to use, and sometimes riders just want the newest app.
 
-## GTFS can probably do more than you think
+## Maintained by the community
 
-GTFS is most widely known for trip planning information, particularly in metro areas with fixed-route service. However, there are a variety of optional features above and beyond the basic GTFS Schedule specification that make GTFS more widely applicable, including Fares for showing fare costs and structures, Flex (in development) for demand-responsive transit options, such as dial-a-ride and paratransit services, and Pathways for displaying accessibility information that’s vital to riders using mobility devices or needing additional accommodations. GTFS Realtime builds upon GTFS Schedule and on-vehicle GPS systems to provide real-time updates on vehicle locations.
+As an open data standard, GTFS is driven by an open community where anyone can propose changes or improvements. The specification continues to evolve and is overseen by an independent nonprofit organization called [MobilityData](https://mobilitydata.org/) guided by the following characteristics:
 
-## GTFS is more than just trip planning
+- Simple: It’s meant to be easy to view and edit using simple tools and editors.
```

Comment:
This seems a little duplicative of the beginning of the doc, could we add "Stable" and "User-centered" in the list above?

Reply:
I'm having second thoughts about this one, I do agree that it might look a bit repetitive, but basically the idea for the first list was to set the structure for the rest of the text. As you can see, Simple, Flexible, Open-Source and Community-Driven are linked with the four subtitles of this page. 

For this list, I was presenting a simplified version of the guiding principles from the Spec Amendment Process. As a fix I updated the list to match the guiding principles more closely. LMK what you think!